test/fixtures/write-index/defaultName/index.js

```diff
@@ -0,0 +1,5 @@
+// @create-index
+
+export { default } from './defaultName.js';
```

Comment:
`defaultName` is confusing for the purpose of the test.

Add a sub-directory `bar` within `defaultName` and contain the fixture in `bar` directory.